.github/workflows/build-verification.yml

```diff
@@ -1,10 +1,8 @@
 name: Verify Build
 
 on:
-  push:
-    branches-ignore:
-      - main
   pull_request:
+    branches: [ main ]
```

Comment:
I know this is the pattern used across the Solutions repos, but it isn't clear to me why we would only want to run build-verification only on PRs targeting `main`.  Is there a downside to running it on all PRs regardless of the branch they target?

Reply:
I am going to merge this PR as is for now, but this is a great point. I think this warrants a larger team discussion.